python/ray/worker.py

```diff
@@ -367,18 +366,17 @@ def put_object(self, object_id, value):
             # should return an error code to the caller instead of printing a
             # message.
             logger.info(
-                "The object with ID {} already exists in the object store."
-                .format(object_id))
+                "The object with ID {} already exists in the object store.",
```

Comment:
I don't think this actually works. I think you need `%s` instead of `{}`.

`import ray; ray.init(); ray.worker.logger.info("asdf {}", 3)` crashes for me. Does this mean that this line is not tested?